frameworks/cassandra/build.sh

```diff
@@ -9,7 +39,12 @@ ROOT_DIR="$(dirname "$(dirname $FRAMEWORK_DIR)")"
 export TOOLS_DIR=${ROOT_DIR}/tools
 BUILD_DIR=$FRAMEWORK_DIR/build/distributions
 PUBLISH_STEP=${1-none}
-${ROOT_DIR}/gradlew distZip -p ${ROOT_DIR}/sdk/executor
+
+if [ x"$cli_only" = xtrue ]; then
```

Comment:
This can just be:
```
if $cli_only; then
```
E.G. https://github.com/mesosphere/tweeter/blob/master/cli_script.sh#L270